### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Teicu** [crazywifi/TOR_Proxy_Burp_Extension#1]
Hi, 

What about macOS. Does it work? Given that there is no Tor binary for Mac 

Thanks

**Comment by crazywifi** [comment]
No, not for Mac :/ . I will research for Mac and will let you know.
